intranet/utils/date.py

```diff
@@ -26,3 +26,11 @@ def get_date_range_this_year(now=None):
         date_start = datetime.datetime(now.year, settings.YEAR_TURNOVER_MONTH + 1, 1, 0, 0, 0)
         date_end = datetime.datetime(now.year + 1, settings.YEAR_TURNOVER_MONTH, 1, 0, 0, 0)
     return timezone.make_aware(date_start), timezone.make_aware(date_end)
+
+
+def get_senior_graduation_year(*, now=None):
+    return get_date_range_this_year(now=now)[1].year
+
+
+def get_senior_graduation_date():
+    return datetime.datetime(year=get_senior_graduation_year(), month=6, day=18, hour=19)
```

Comment:
I don't think the day/time of graduation should be hardcoded.

Reply:
Should these be set in `settings` instead?

Reply:
Yes